### ISSUE TITLE
Support parrot cargo ?

### ISSUE THREAD

**Issue by marianodominguez** [voodootikigod/node-rolling-spider#70]
According to doc, the difference with rolling spider is the lego pieces, wonder if software is too different.

I was able to connect to device, but it seems it does not hold naming convention:

`
  rollingspider Peripheral { _noble: Noble { state: 'poweredOn', _bindings: EventEmitter { _state: 'poweredOn', _addresses: [Object], _addresseTypes: [Object], _connectable: [Object], _pendingConnection: false, _connectionQueue: [], _handles: {}, _gatts: {}, _aclStreams: {}, _hci: [Object], _gap: [Object], _events: [Object], _eventsCount: 20, onSigIntBinded: [Function: bound ], _scanServiceUuids: [] }, _peripherals: { '68644b1f30cc': [Object], e014e7343da3: [Circular] }, _services: { '68644b1f30cc': {}, e014e7343da3: {} }, _characteristics: { '68644b1f30cc': {}, e014e7343da3: {} }, _descriptors: { '68644b1f30cc': {}, e014e7343da3: {} }, _discoveredPeripheralUUids: [ '68644b1f30cc', 'e014e7343da3' ], _events: { warning: [Object], discover: [Function: bound ], stateChange: [Function: bound ] }, _eventsCount: 3, _allowDuplicates: undefined }, id: 'e014e7343da3', uuid: 'e014e7343da3', address: 'e0:14:e7:34:3d:a3', addressType: 'random', connectable: true, advertisement: { localName: 'Travis_050419', txPowerLevel: undefined, manufacturerData: <Buffer 43 00 cf 19 09 09 01 00>, serviceData: [], serviceUuids: [] }, rssi: -59, services: null, state: 'disconnected' } +0ms
  rollingspider Travis_050419 +3ms
`

**Comment by sandeepmistry** [comment]
@marianodominguez the manufacturer data does match the check here: 
https://github.com/voodootikigod/node-rolling-spider/blob/master/lib/drone.js#L94

Once you connect can you take off? See [keyboard.js](https://github.com/voodootikigod/node-rolling-spider/blob/master/eg/keyboard.js) example.

If that works, a similar PR to #64 could be added to support it.

**Comment by marianodominguez** [comment]
Hello, I think the problem is the local name not starting with "RS_",  so it is not recognized as drone, .connect() just ignore the entry and waits forever.

I'll test removing the restriction on name.

**Comment by virgilvox** [comment]
I just put up a pull request for commit #71 

@marianodominguez your suggestion worked :+1:

**Comment by marianodominguez** [comment]
@virgilvox added my model to pull request #72 ... my drone is taking off :)

**Comment by sandeepmistry** [comment]
@marianodominguez @virgilvox awesome, great news!

It would be good to also add checks to match on the manufacturer data value, as the local name can be changed via the Parrot mobile app. (See #26)

**Comment by marianodominguez** [comment]
@sandeepmistry manufacturer id does not match, either : for my Travis is 

`manufacturerData: <Buffer 43 00 cf 19 09 09 01 00>`
`manufacturer.toString('hex')`

'4300cf1909090100' 

yours is 

'4300cf1900090100'

@virgilvox which mfr does your mars retrieve?

**Comment by virgilvox** [comment]
@marianodominguez for Mars_ its 4300cf1909090100

**Comment by Frostlock** [comment]
I stumbled on the same thing for the Airborne night swat drone.

The manufacturer check is not triggering for my drone
I'm very new to nodejs so I might be missing something.
`console.log(manufacturer)`
shows
`<Buffer 43 00 cf 19 07 09 01 00>`

For some reason this does not trigger the check on 
`&& (['4300cf1900090100', '4300cf1909090100', '4300cf1907090100'].indexOf(manufacturer) >= 0);`

My workaround :)
I renamed my drone earlier through freeflight3 on ipad. So I adapted the name check to check for my drones name which worked.
